gradient/cli/common.py

```diff
@@ -171,3 +219,31 @@ def options_file(f):
         )
     ]
     return functools.reduce(lambda x, opt: opt(x), reversed(options), f)
+
+class GradientObjectListOption(GradientOption):
+    def __init__(self, object_name, param_decls, **kwargs):
+        super(GradientObjectListOption, self).__init__(param_decls, **kwargs)
+        self.object_name = object_name
+        self.object_list_name = self.object_name + "s"
+        self.object_key = self._compose_object_key()
+
+    # Get name of overall object list for this type of option
+    def get_object_list_name(self):
+        return self.object_list_name
```

Comment:
No need to define getters in python like that. If there is a need to change reading object's field  value to a function, it can be replaced with a property. The field is public anyway so this function is not needed and can be removed

https://docs.python.org/3/library/functions.html#property